### PR TITLE
RUM-4738: Add "Tab" and "TabRow" Composable groups mappers

### PR DESCRIPTION
**Pull request by ambushwork**
### What does this PR do?

Add "Tab" and "TabRow" Composable groups mappers, so that "Tab" & "TabRow" can have:
* correct position in SR
* correct color in SR
* correct size in SR

### Motivation

RUM-4738

### Demo

(The other components in the screen is a bit glitchy, but `Tab` and `TabRow` is working)

https://mobile-integration.datadoghq.com/rum/replay/sessions/0d175d4a-4087-4cc9-8abc-fe22ee7d3c0e?applicationId=38030dde-f9f9-4e52-9443-b9804a030080&seed=e05a16d0-ea37-4f5b-b4fd-2ff7f27fe3fb&ts=1723037890870

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

File: features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/TabCompositionGroupMapper.kt [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                x = boxWithDensity.x,
+                y = boxWithDensity.y,
+                width = boxWithDensity.width,
+                height = boxWithDensity.height
+            ),
+            uiContext.copy(parentContentColor = textColor)
+        )
+    }
+
+    private fun parseContentColor(isSelected: Boolean, param: ComposableParameter): String? {
```
Comment (mariusc83) [inline features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/TabCompositionGroupMapper.kt:45]: this looks a bit weird ...why not having 2 functions...one that returns the `selectedColor` and one the `unselectedColor` and based on the `isSelected` value you call one or the other ?

File: features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/TabRowCompositionGroupMapper.kt [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+) : AbstractCompositionGroupMapper(colorStringFormatter) {
+    override fun map(
+        stableGroupId: Long,
+        parameters: Sequence<ComposableParameter>,
+        boxWithDensity: Box,
+        uiContext: UiContext
+    ): ComposeWireframe {
+        var backgroundColor: String? = null
+        var contentColor: String? = null
+        parameters.forEach { param ->
```
Comment (mariusc83) [inline features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/TabRowCompositionGroupMapper.kt:27]: isn't the case where you can have either the `backgroundColor` and `contentColor` available in the same time ?

File: features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/TabCompositionGroupMapper.kt (hunk 1 above)
Comment (ambushwork) [reply features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/TabCompositionGroupMapper.kt]: you are right, updated

File: features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/TabRowCompositionGroupMapper.kt (hunk 2 above)
Comment (ambushwork) [reply features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/TabRowCompositionGroupMapper.kt:27]: I am not sure what you mean here, do you have any suggestion?

Comment (mariusc83) [reply features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/TabRowCompositionGroupMapper.kt:27]: nevermind, i didn't see it well below :)

**Review by mariusc83 — APPROVED** [review]

**Pull request merged**